tests/baselines/reference/recursiveConditionalTypes.errors.txt

```diff
@@ -85,8 +84,6 @@ tests/cases/compiler/recursiveConditionalTypes.ts(116,9): error TS2345: Argument
     type TT2 = TupleOf<number, number>;
     type TT3 = TupleOf<number, any>;
     type TT4 = TupleOf<number, 100>;  // Depth error

```

Comment:
The comment in `tests/cases/compiler/recursiveConditionalTypes.ts` will need to be amended.